### PR TITLE
Add Dimensions and Image types

### PR DESCRIPTION
**Pull request by michaelcfine**
- Added Dimensions type
- Added Image type
- Note: Both of these are for use in the upcoming github.com/rdkcentral/firebolt-configurations repo and its definition of schemas for the app manifest file.

### CODE REVIEW

File: src/schemas/types.json [hunk 1]
```diff
@@ -160,6 +160,50 @@
...
+                    "type": "integer",
+                    "minimum": 1
+                }
+            },
+            "required": [ "width", "height" ]
+        },
+        "Image": {
+            "type": "object",
+            "properties": {
+                "path": {
```
Comment (jlacivita) [inline src/schemas/types.json:181]: ideally we could reuse this for both web and packaged images. Can we call this `uri` which is appropriate for both relative paths, e.g. `./foo/image.png` and also hosted images, e.g. `https://foo.com/bar.png`

**Review by jlacivita — CHANGES_REQUESTED** [review]
Looks good except I'd like to discuss `path`

File: src/schemas/types.json [hunk 2]
```diff
@@ -160,6 +160,50 @@
...
+            "properties": {
+                "uri": {
+                    "description": "URI for the image. May be a relative path (e.g. ./foo/image.png) or absolute (e.g. https://foo.com/bar.png) depending on usage.",
+                    "type": "string"
+                },
+                "aspectRatio": {
+                    "description": "The aspect ratio of the image",
+                    "type": "string",
+                    "enum": [
+                        "1x1", "4x3", "16x9", "2x3", "4x5"
```
Comment (kpears201) [inline src/schemas/types.json:189]: Seems odd this is an enumeration, we should support any aspect ratio right?

Comment (michaelcfine) [reply src/schemas/types.json:189]: I can change this to just use a regex to confirm that it is of the form "<int>:<int>". That work?

**Pull request closed without merging**